Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -963,8 +988,82 @@ def get_mapping_fields_command(service):
     demisto.results(types_map)
 
 
+def get_last_update_in_splunk_time(last_update):
+    """ Transforms the time to the corresponding time on the Splunk server
+
+    Args:
+        last_update: The time to be transformed
+
+    Returns: The corresponding time on the Splunk server
+
+    """
+    last_update_utc_datetime = dateparser.parse(last_update, settings={'TIMEZONE': 'UTC'})
+    dem_params = demisto.params()
+    splunk_timezone = dem_params.get('timezone')
+    if not splunk_timezone:
+        raise Exception('Cannot mirror incidents when timezone is not configured. Please enter the '
+                        'timezone of the Splunk server being used in the integration configuration.')
+    return date_to_timestamp(last_update_utc_datetime + timedelta(minutes=int(splunk_timezone))) / 1000
+
+
+def get_remote_data_command(service):
+    notable_updated_data = []
+    entries = []
+    updated_notable = {}
+    remote_args = GetRemoteDataArgs(demisto.args())
+    last_update_splunk_timestamp = get_last_update_in_splunk_time(remote_args.last_update)
+    notable_id = remote_args.remote_incident_id
+    search = '|`incident_review` ' \
```

Comment:
what does `incident_review` do ?

Reply:
A notable can change in splunk using the Incident Review tab, mostly changes of user assignment, owner change, comment addition, status change.

The \`incident_review\` macro fetches all notable that have changed.